proposals/semantic_schema/README.md

```diff
@@ -133,13 +133,45 @@ string[] semantics:instance_name:labels = ["class:animal", "class:bird", "class:
 
 The proposed method has better performance as the number of labels of the same type increase on a prim to dozens, hundreds, or thousands of labels.
 
+### Semantic Aggregation
+
+"Aggregating" semantic data is essential to workflows.  A prim could have the semantic label of `door`, but that is missing wider context.  Is the prim a door inside of a room, on the exterior of a building, part of a vehicle?
+
+A method should be provided to "aggregate" parent semantics in a consistent way that allows users to understand the semantic heirarchy.
+
+Example prim setup:
+```
+└── xform - car
+    ├── wheel
+    ├── door
+    │   ├── window(glass)
+    │   └── handle
+    └── seat
+```
+Querying the semantic heirarchy should return an appended list/set of parents
```

Comment:
to make it more blindingly explicit, maybe something like:

... should return a uniquified list of labels, where ordering is determined by ancestral distance from the prim being queried, and the _first_ time a label is encountered in the ancestors-walk determines its position in the list.